pkg/pdapi/pdetcd.go

```diff
@@ -19,11 +19,13 @@ import (
 	"time"
 
 	etcdclientv3 "github.com/coreos/etcd/clientv3"
+	etcdclientv3util "github.com/coreos/etcd/clientv3/clientv3util"
 )
 
-type PDEtcdApi interface {
+type PDEtcdClient interface {
+	// PutKey would put key to the target pd etcd cluster
```

Comment:
why use `would` here? At the end of the function, the put/delete operation may not finish?

Reply:
I think the comment is misleading, I should use `will` here.